dali/pipeline/executor/executor2/exec2.cc

```diff
@@ -91,6 +92,7 @@ class Executor2::Impl {
   };
 
   void Build(const graph::OpGraph &graph) {
+    DomainTimeRange tr("[DALI][Executor] Build");
```

Comment:
How about omitting the `[DALI]` tag, since the Domain is already named DALI?
```suggestion
    DomainTimeRange tr("[Executor] Build");
```

Reply:
It may have a merit - but then I guess we should revisit all NVTX markers and drop DALI, otherwise this would stand out.